tests/functional/python_model_tests/test_list_inference.py

```diff
@@ -0,0 +1,36 @@
+"""
+This test case addresses this regression: https://github.com/dbt-labs/dbt-bigquery/issues/1047
+
+As the comments point out, the issue appears to be that the default settings are:
```

Comment:
this comment is true but won't be post merge. Maybe rephrase to say "this issue appears if/when"

Reply:
I updated the comment as suggested, and made a few other updates since this comment went out of date with what was done below (e.g. `files` instead of `models`).